### PR TITLE
v5.3 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/administration/changelog.md [hunk 1]
```diff
@@ -4,6 +4,83 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+- Added username and profile pictures to Outgoing Webhooks setup page.
+- Added "Deactivate Account" option under Account Settings > Advanced.
+- Added member count for the Direct Messages list.
+- Added support for attachments import to Direct Message channels.
+
+#### Performance
+- Added short delay before querying server in suggestion lists to prevent spamming.
+
+#### Administration
+- Created an telemetry event for when telemetry is turned off from the System Console.
```
Comment (ThiefMaster) [inline source/administration/changelog.md]: grammar, should be "*a* telemetry event"

**Comment by jasonblais** [comment]
Overall nice work 👍

**Comment by amyblais** [comment]
@cpanato To update API / Websocket / Database sections

**Comment by cpanato** [comment]
@amyblais nothing to add

**Pull request merged**